.github/workflows/create-lint-wf.yml

```
@@ -1,12 +1,15 @@
 name: Create a pipeline and lint it
 on: [push, pull_request]
 
-# Uncomment if we need an edge release of Nextflow again
-# env: NXF_EDGE: 1
-
 jobs:
   MakeTestWorkflow:
     runs-on: ubuntu-latest
+    env:
+      NXF_ANSI_LOG: false
+    strategy:
+      matrix:
+        # Nextflow versions: check pipeline minimum and latest edge version
+        nxf_ver: ["NXF_VER=21.04.0", "NXF_EDGE=1"]
```

Do we need to specify the version? If not specified it just pulls the latest stable, right? In which case maybe it's easier to just specify edge or not:

```suggestion
        nxf_edge: [0, 1]
```

---

Yep, that would work too but I thought the idea with specifying the version was more to check the mimimum version specified in the pipeline template is working too?

---

Because that won't always be the same as the latest stable version.

---

Or maybe it will if we have 2 NF releases per yer 🤷🏽 😆 Don't mind either way.

---

ah ok, so minimum plus bleeding edge - yeah sounds good then 👍🏻

---

Maybe we should do the same thing in the pipeline template CI 🤔